dev/plugins/hu.elte.txtuml.export.cpp/src/hu/elte/txtuml/export/cpp/thread/ThreadDescriptionExporter.java

```diff
@@ -150,4 +135,23 @@ private void checkEmptyGroup(Class<? extends ModelClass>[] classes) {
 		}
 
 	}
+
+	private void checkConfigurationOptions(double gradient, int contstant, int max) {
+		if (gradient < 0 || gradient > 1) {
+			warningList.add("The gradient of linear function should be beetween 0 and 1: " + "conversion to zero.");
```

Comment:
Typo, should be `the linear function`, `between` and `0` (instead of `zero`, for consistency).